x-pack/packages/kbn-cloud-security-posture/src/hooks/use_csp_setup_status_api.ts

```diff
@@ -6,16 +6,18 @@
  */
 
 import { useQuery, type UseQueryOptions } from '@tanstack/react-query';
-import { useKibana } from '../hooks/use_kibana';
-import { type CspSetupStatus } from '../../../common/types_old';
-import { STATUS_API_CURRENT_VERSION, STATUS_ROUTE_PATH } from '../../../common/constants';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
```

Comment:
Originally I did that but there seems to be an issue when I moved our useKibana to the package (things starts to fail) and separate from that issue I also had a discussion with @opauloh where we think that it might be better to have 2 different useKibana due to plugin dependencies

Reply:
let's see if we can bring it back as a part of a smaller PR separately so it's easier to reason about. If not, using the hook directly from `@kbn/kibana-react-plugin/public` is fine, we might just need to create a type to not repeat `CoreStart & CspClientPluginStartDeps` all the time